### PR TITLE
allow for extra plugins specified by module path (vs file path)

### PR DESCRIPTION
**Pull request by herb**
feedback welcome but dependent upon https://github.com/gmjosack/annex/pull/2, please don't merge yet.

### CODE REVIEW

**Comment by rra** [comment]
The `bin/*c` compiled versions seem to keep ending up in diffs.  I think we have a missing `.gitignore` entry, maybe?

**Comment by herb** [comment]
indeed. added `3920b6d` to update `.gitignore` and to remove the committed `*-apic`

File: grouper/fe/settings.py [hunk 1]
```diff
@@ -15,16 +15,16 @@ def override_timezone(self, value):
...
-    "date_format": "%Y-%m-%d %I:%M %p",
-    "cdnjs_prefix": "//cdnjs.cloudflare.com",
-    "user_auth_header": "X-Grouper-User",
     "refresh_interval": 60,
-    "url": "http://127.0.0.1:8888",
-    "how_to_get_help": None,
     "site_docs": None,
+    "timezone": FeSettings.default_timezone,
+    "url": "http://127.0.0.1:8888",
+    "user_auth_header": "X-Grouper-User",
```
Comment (lfaraone) [inline grouper/fe/settings.py]: Are these changes related to this PR?

Comment (herb) [inline grouper/fe/settings.py]: strictly 'no'. i just alphabetized the list so i could find stuff.

Comment (lfaraone) [inline grouper/fe/settings.py]: OK :) It'd be awesome if this could be broken out into a separate commit so future folks don't get confused :)

Comment (herb) [inline grouper/fe/settings.py]: ok. done.

**Comment by lfaraone** [comment]
hey @herb -- is this ready to be re-reviewed? I see the PR you referenced in the description has been merged.

**Comment by cbguder** [comment]
Closing this as we merged #338
